source/macros.tex

```diff
@@ -393,9 +393,9 @@
 
 \newenvironment{bnfbase}
  {
- \newcommand{\nontermdef}[1]{\nonterminal{##1}\indexgrammar{\idxgram{##1}}:}
+ \newcommand{\nontermdef}[1]{\nonterminal{##1}\indexgrammar{\idxgram{##1}}\textnormal{:}}
```

Comment:
Yeah, I had the same idea. And adding \/ or \itcorr indeed fixes it. Screenshot forthcoming.